fcl/dunefd/reco/standard_reco2_atmos_dune10kt_1x2x6.fcl

```diff
@@ -0,0 +1,24 @@
+#include "angularreco.fcl"
+#include "standard_reco2_dune10kt_1x2x6.fcl"
+
+physics.producers.pandora.ConfigFile:   "PandoraSettings_Master_Atmos_DUNEFD.xml"
+
+physics.producers.cvnmap: @local::dunehd_atmo_1x2x6_cvnmapper
+physics.producers.cvneva: @local::dunehd_1x2x6_atmo_cvnevaluator
+
+#Using the highest energy shower direction
+physics.producers.anglereconue:  @local::dunefd_nuangularreco_pandora_nue
+#Using the longest track direction
+physics.producers.anglereconumu: @local::dunefd_nuangularreco_pandora_numu
+#Using all the reconstructed PFPs
+physics.producers.anglereconuepfps: @local::dunefd_nuangularreco_pandora_nue_allpfps
+#Using all the reconstructed PFPs assuming longest track is a muon
+physics.producers.anglereconumupfps: @local::dunefd_nuangularreco_pandora_numu_allpfps
```

Comment:
I think these definitions should go in the workflow fcl in the list of producers: https://github.com/AndyChappell/dunesw/blob/feature/atmos_prod_phase2/fcl/dunefd/reco/workflow_reco_dune10kt.fcl